tools/mo/openvino/tools/mo/main.py

```diff
@@ -352,6 +393,13 @@ def prepare_ir(argv : argparse.Namespace):
             t.send_event("mo", "conversion_method", moc_front_end.get_name() + "_frontend")
             moc_front_end.add_extension(TelemetryExtension("mo", t.send_event, t.send_error, t.send_stack_trace))
             moc_front_end.add_extension(ProgressReporterExtension(progress_printer(argv)))
+            # after check_fallback call only new kind of extensions and transformations_config can processed here
+            if hasattr(argv, 'transformations_config') and argv.transformations_config is not None and len(argv.transformations_config):
+                moc_front_end.add_extension(JsonConfigExtension(argv.transformations_config))
+            if hasattr(argv, 'extensions') and argv.extensions is not None and len(argv.extensions) > 0 \
+                and argv.extensions != import_extensions.default_path():
+                for extension in argv.extensions.split(','):
+                    moc_front_end.add_extension(extension)
             ngraph_function = moc_pipeline(argv, moc_front_end)
             return graph, ngraph_function
         else: # apply fallback
```

Comment:
What will be if an user pass legacy `extension` or `transformation_config` in case of converting PDPD model?
I think we need to check return values from `deduce_framework_by_namespace` and raise error if all of them are `False`

Reply:
handled such scenario, but without `deduce_framework_by_namespace`